guides/source/active_storage_overview.md

```diff
@@ -32,6 +32,26 @@ Using Active Storage, an application can transform image uploads or generate ima
 representations of non-image uploads like PDFs and videos, and extract metadata from 
 arbitrary files.
 
+### Requirements
+
+Various features of Active Storage depend on third party applications which Rails 
+will not install, and must be included separately.
+
+* [ImageMagick](https://imagemagick.org/index.php) or [libvips](https://github.com/libvips/libvips) v8.6+ 
+for image analysis and transformations;
+* [ffmpeg](http://ffmpeg.org/) v3.4+ for video/audio analysis, and video previews;
+* [poppler](https://poppler.freedesktop.org/) or [muPDF](https://mupdf.com/) for PDF previews;
+
+To enable image transformations and analysis the `image_processing` gem is also necessary:
+
+```ruby
+gem 'image_processing', '>= 1.2'
+```
+
+TIP: ImageMagick is the better known and more widely available of the two applications. However, libvips [is known](https://github.com/libvips/libvips/wiki/Speed-and-memory-use) to be up to 10x faster and consume 1/10 the memory. For JPEG files this can be further improved by removing `libjpeg-dev` and installing `libjpeg-turbo-dev` in its place. The latter is 2-7x faster then the former. 
```

Comment:
```suggestion
TIP: Compared to libvips, ImageMagick is better known and more widely available. However, libvips can be [up to 10x faster and consume 1/10 the memory](https://github.com/libvips/libvips/wiki/Speed-and-memory-use). For JPEG files, this can be further improved by replacing `libjpeg-dev` with `libjpeg-turbo-dev`, which is [2-7x faster](https://libjpeg-turbo.org/About/Performance). 
```

Reply:
@ghiculescu Yes, the first half could be removed if it does (the `libjepg-turbo` part is still useful). I was not sure if it would however. Should I remove it so the PR can be merged or do we wait to see what will happend to #42744 before merging this one?

Reply:
I think we can go with what we have, and then rebase #42744 to remove any documentation that's no longer applicable.